src/ansys/fluent/core/session_solver.py

```diff
@@ -246,29 +264,26 @@ def read_case_lightweight(self, file_name: str):
 
     def get_state(self) -> StateT:
         """Get the state of the object."""
-        return self._root.get_state()
+        return self.root.get_state()
 
     def set_state(self, state: Optional[StateT] = None, **kwargs):
         """Set the state of the object."""
-        self._root.set_state(state, **kwargs)
+        self.root.set_state(state, **kwargs)
 
     def __call__(self):
         return self.get_state()
 
     def _populate_settings_api_root(self):
         if not self._settings_api_root:
-            self._settings_api_root = _import_settings_root(self._root)
+            self._settings_api_root = _import_settings_root(self.root)
```

Comment:
👍  Correctly made this public. The next level of thinking is that the name is actually a bit vague. How about `settings`?
```
>>> state = solver.settings.get_state()
>>> settings = solver.settings
>>> settings.setup.models...
```

Reply:
N.b. in the (new) console right now:
```
>>> import ansys.fluent.core.solver.flobject
>>> G_ = ansys.fluent.core.solver.flobject.Group
>>> [x for x in dir(solver) if isinstance(getattr(solver, x), G_)]  
['_root', '_settings_root', 'design', 'file', 'mesh', 'parallel', 'parameters', 'results', 'server', 'setup', 'solution']
>>> 
```
Both '_root' and '_settings_root' are in the list separately.  We only would like `settings` in both PyFluent & PyConsole.

@mkundu1 @prmukherj

Reply:
@seanpearsonuk Are we thinking to supersede `solver.setup` by `solver.settings.setup`? I agree that the grouping all top level settings object under a common head is better, but we need to plan the corresponding Python journal recording change in Fluent.

Reply:
n.b. agreed to keep both formats in 242 intending to deprecate solver.setup etc in 251 and update the journaling